dlp/snippets/trigger/list.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trigger
+
+// [START dlp_list_triggers]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"google.golang.org/api/iterator"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// listTriggers lists the triggers for the given project.
+func listTriggers(w io.Writer, projectID string) error {
+	// projectID := "my-project-id"
+
+	ctx := context.Background()
+
+	client, err := dlp.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("dlp.NewClient: %v", err)
+	}
+
+	// Create a configured request.
+	req := &dlppb.ListJobTriggersRequest{
+		Parent: "projects/" + projectID,
+	}
+	// Send the request and iterate over the results.
+	it := client.ListJobTriggers(ctx, req)
+	for {
+		t, err := it.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			return fmt.Errorf("Next: %v", err)
+		}
+		c := t.GetCreateTime()
+		u := t.GetUpdateTime()
```

Comment:
maybe move directly above where they're used?

Reply:
and maybe want to use ptypes?

https://godoc.org/github.com/golang/protobuf/ptypes#Timestamp

Reply:
Done and done. Didn't know about that method - thanks.